flexmeasures/ui/templates/views/sensors.html

```diff
@@ -33,6 +33,58 @@
               <div class="alert alert-info" id="dstwarn" style="display:none;"></div>
               <div class="alert alert-info" id="sourcewarn" style="display:none;"></div>
           </div>
+          <div class="container">
+          <div class="row">
+          <div class="col-sm-6">
+                    <h3>Stats</h3>
+                    <table id="statsTable" class="table table-striped">
+                        <tbody id="statsTableBody">
```

Comment:
Can we display a spinner inside the table until data is loaded or if that is difficult, the word "loading ..."?

Reply:
Can you check if I have used the spinner correctly?